style.css

```diff
@@ -511,3 +528,155 @@ textarea:focus,
 input:focus {
   outline: none;
 }
+
+.body-sec {
+  background-color: #fff;
+}
+
+@media screen and (min-width: 768px) {
+  .mobile-menu {
+    display: none;
+  }
+
+  .desktop-menu {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    padding-right: 16px;
+  }
+
+  .menu-items {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    gap: 16px;
+  }
+
+  .menu-item a {
+    font-family: "Poppins", sans-serif;
+    font-style: medium;
+    font-weight: 500;
+    font-size: 15px;
+    line-height: 20px;
+    color: #344563;
+    margin-right: 16px;
+    margin-left: 16px;
+    padding: 10px;
+  }
+
+  .home {
+    background-image: url(./image/Desktop-Images/desktopbg.svg);
+    justify-content: first baseline;
+    background-size: cover;
+  }
+
+  .title {
+    font-style: bold;
+    font-size: 48px;
+    width: 100%;
+    line-height: 80px;
+  }
+
+  .text {
+    font-size: 20px;
+    line-height: 28px;
+    margin-top: 16px;
+    padding-top: 15px;
+  }
+
+  .connect {
+    margin-top: 12px;
+  }
+
+  .card {
+    flex-direction: row;
+    justify-content: center;
+    align-items: center;
+    margin: 16px 100px;
+    height: 60vh;
+  }
```

Comment:
- [ ] *[Optional]* I think it would be better to adjust the padding of your cards to replicate the Figma design, currently the padding is bigger than expected 😎😎😎

   It is also possible to make the project's Images dynamic in width and height by adding to the **img** selector ```object-fit: cover;```, this will help us to have a **responsive image** 👍 

**Currently**
![image](https://user-images.githubusercontent.com/102757126/222877950-9a28bcca-985a-4697-afe8-e7283e0cfe69.png)

**Figma design**
![image](https://user-images.githubusercontent.com/102757126/222878018-d45899ce-6486-4579-9ffd-6d258d93d408.png)